src/main/menu.js

```diff
@@ -47,6 +48,8 @@ export const setupMenu = store => {
           type: 'separator'
         },
         {}, // Import menu will be injected here
+        {}, // Import search menu seperator
```

Comment:
You can just directly put the menu items here.  You don't need to update them later. That is for the cases where that menu is dynamic